static/downloads/influxdb-k8-minikube.yaml

```diff
@@ -23,7 +23,7 @@ spec:
                 app: influxdb
         spec:
             containers:
-              - image: quay.io/influxdb/influxdb:v2.0.1
+              - image: quay.io/influxdb/influxdb:v2.0.3
```

Comment:
We'll need to be sure to update this once 2.0.4 goes out. Is there a `latest` tag we could use so we don't have to remember to update this on each release?

Reply:
Does not appear to be a `latest` atm: https://quay.io/repository/influxdb/influxdb?tag=v2.0.3&tab=tags.

Reply:
K, will you update this on the 204-changelog branch for the 2.0.4 release?